coalib/bears/Bear.py

```diff
@@ -12,8 +12,10 @@
 from coala_utils.decorators import (enforce_signature, classproperty,
                                     get_public_members)
 
+# Ignore PyUnusedCodeBear
 from coalib.bears.requirements.PackageRequirement import PackageRequirement
-from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.bears.requirements.PipRequirement import PipRequirement \
+                                                    # Ignore PyUnusedCodeBear
```

Comment:
stick the comment above..... it looks cleaner. Alternatively, since you have a few lines that need ignoring, you might want to use the `start ignoring` `stop ignoring sytax`,(see https://docs.coala.io/en/latest/Users/Tutorial.html#ignoring-code-inside-files)